setup.cfg

```diff
@@ -57,7 +57,7 @@ universal = 1
 [flake8]
 max-line-length = 88
 select = C,E,F,W,B
-ignore = E203, W503
+ignore = E203, W503, F401
```

Comment:
Where is F401 triggered?

Reply:
`typing` members that only appear in comments.

Reply:
Ah OK.